VideoWeb/VideoWeb.Contract/Responses/ClientSettingsResponse.cs

```diff
@@ -14,25 +15,6 @@ public class ClientSettingsResponse
         /// The eventhub path
         /// </summary>
         public string EventHubPath { get; set; }
-        /// <summary>
```

Comment:
Good call, I noticed we don't need or use this in video web when setting up the docker images